packages/RNTester/js/examples/Alert/AlertExample.js

```diff
@@ -152,7 +184,57 @@ const styles = StyleSheet.create({
   },
 });
 
-module.exports = {
-  AlertExample,
-  SimpleAlertExampleBlock,
-};
+exports.title = 'Alerts';
+exports.description =
+  'Alerts display a concise and informative message ' +
+  'and prompt the user to make a decision.';
```

Comment:
how about just making them a single string ?

Reply:
It was already like that before. I think the idea behind is to wrap the text so it can be read without scrolling. I am not sure If I should change it.

Reply:
No, I've seen this in other places in the code. This is the convention when including long strings, this is good as is